### ISSUE TITLE
Needs a label for the percent number

### ISSUE THREAD

**Issue by Azzurite** [andrey-zakharov/roicalc#3]
I have no idea what it means.

**Comment by andrey-zakharov** [comment]
Yes, this is demand in shop. Actually, just a modifier for product price to calculate return of investments (at bottom "Summary" row  of table).
